osrm/initialize.py

```diff
@@ -0,0 +1,87 @@
+"""This file contains the commands to spin up an OSRM routing server."""
+import os
+import subprocess
+import sys
+
+
+COMPRESSED_FILENAME = 'california-latest.osm.pbf'
+COMPRESSED_FILEPATH = '/data/osrm/{}'.format(COMPRESSED_FILENAME)
+PROCESSED_FILEPATH = COMPRESSED_FILEPATH.replace('osm.pbf', 'osrm')
+
+
+def _is_file_missing(filepath):
+    """Return True if the file does not already exist."""
+    return not os.path.isfile(filepath)
+
+
+def fetch_osrm_data():
+    """Download the compressed road network files."""
+    steps = [
+        {
+            'name': 'Fetch compressed road network file.',
+            'command':
+                'wget http://download.geofabrik.de/north-america/us/{filename} '
+                ' -O {filepath}'.format(
+                    filename=COMPRESSED_FILENAME,
+                    filepath=COMPRESSED_FILEPATH
+                )
+        }
+    ]
+    exit_code = 0
+    for step in steps:
+        subprocess.call(args=['echo "{}"'.format(step['name'])], shell=True)
+        subprocess.call(args=['echo "{}"'.format(step['command'])], shell=True)
+        exit_code = subprocess.call(args=[step['command']], shell=True)
+        if exit_code != 0:
+            sys.exit(exit_code)
+
+
+def process_osrm_data():
```

Comment:
Should COMPRESSED_FILENAME and such be input instead?

Reply:
Done.